src/lib.rs

```diff
@@ -682,6 +732,43 @@ impl LightningNode {
         }
     }
 
+    /// Pay an LNURL-pay the provided amount.
+    ///
+    /// Parameters:
+    /// * `amount_sat` - amount to be paid
+    /// * `lnurl_pay_request_data` - LNURL-pay request data as obtained from [`LightningNode::decode_data`]
+    ///
+    /// Returns the payment hash of the payment.
+    pub fn lnurl_pay(
```

Comment:
Rename to `pay_lnurl` or `pay_lnurlp`

Reply:
Applied in [aafd8e7](https://github.com/getlipa/lipa-lightning-lib/pull/711/commits/aafd8e71624b20652d8a977f4bda9faebe34d445)